crates/astria-conductor/src/conductor.rs

```diff
@@ -319,6 +314,49 @@ impl Conductor {
     }
 }
 
+async fn get_executor(
```

Comment:
```suggestion
async fn make_executor(
```
Or alternatively `create_` or smth like that. Reason: `get_*` as a free standing function feels like you pulling some object from global context (located at some `static` memory location).